pkg/templates/nstemplatetiers/nstemplatetier_generator_whitebox_test.go

```diff
@@ -572,18 +595,11 @@ func assertNamespaceTemplate(t *testing.T, decoder runtime.Decoder, actual templ
 		default:
 			t.Errorf("unexpected kind: '%s'", kind)
 		}
+	case "appstudio":
+		// Nothing
```

Comment:
all the networkpolicies but `allow-from-other-user-namespaces` are still there - why shouldn't we check them as well?